src/ledger-bridge.ts

```diff
@@ -27,13 +25,17 @@ export type LedgerSignTypedDataResponse = Awaited<
 >;
 
 // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
-export interface LedgerBridge {
+export interface LedgerBridge<T extends Record<string, unknown>> {
   isDeviceConnected: boolean;
 
-  init(bridgeUrl: string): Promise<void>;
+  init(): Promise<void>;
 
   destroy(): Promise<void>;
 
+  getOptions(): T;
+
+  setOptions(opts: T): void;
```

Comment:
This method should probably remain async: even if setting the options variable is not per se an async operation, when changing options in the bridge there will most likely be some side effect, and those ones could be asynchronous (eg. when changing bridge URL we should do something to load another iframe).